backend/python/app/routes/entity_routes.py

```diff
@@ -0,0 +1,75 @@
+from flask import Blueprint, request
+from flask import jsonify
+
+from ..resources.entity_dto import EntityDTO
+from ..services import entity_service
+
+# defines a shared URL prefix for all routes
+blueprint = Blueprint('entity', __name__, url_prefix='/entities')
+
```

Comment:
I know you were following the bootcamp example, but for robustness let's handle error checks with try catch instead of null checks